modules/websearch/lib/search_engine.py

```diff
@@ -5017,7 +5017,7 @@ def print_record(recID, format='hb', ot='', ln=CFG_SITE_LANG, decompress=zlib.de
         display_claim_this_paper = False
 
     can_edit_record = False
-    if check_user_can_edit_record(user_info, recID):
+    if  not (format.lower().startswith('t')) and check_user_can_edit_record(user_info, recID):
```

Comment:
if we are seeking for the content of only one field from metadata, no need to check whether user can or not edit record